go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/go-debos/fakemachine
+
+go 1.11
+
+require (
```

Comment:
it'd be nice to note how to generate these autogenerated files?

Reply:
There's a ton of information to unpack here, I'll try to keep it brief:

Most/all notes in the go.sum and go.mod files get discarded on updates - so an inline comment is no-go.
Personally I created the go.mod manually - the dependencies were known from the existing PKGBUILD.

Then created the go.sum via
```
export GOFLAGS="-buildmode=pie -trimpath -mod=mod -modcacherw"
go build ./cmd/fakemachine/main.go # this step downloads the modules and generates go.sum
```

NOTE: With 1.16 (or was it 1.15) the default is `-mod=readonly` - which is recommended for 99% of the uses. Yet since module info is read-only go refuses (hard fails) when updating the go.sum file - hence `-mod=mod`.

In general, the linked documentation suggests using:
```
go mod init github.com/go-debos/fakemachine
go tidy
```
I've avoided that due to two reasons - a) uses `go 1.16` which is not available on Debian and b) uses the latest gocpio symver gocpio tag `v1.0.1`*

* That gocpio is broken - doesn't build. And the latest tag `1.1.0` is not symver (missing leading v) - [fix requested](https://github.com/surma/gocpio/issues/6). Go also refuses the to use the tag and creates it's own pseudo tag/branch. 

All that said - let me know if you want any of this in the commit message,